ee/maintained-apps/outputs/notion/windows.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
@jahzielv can you figure out why this is being added and drop it back out until we have installer URLs?

Reply:
I'm actually in the middle of a change that might make Notion's output legit, so I will try to get that up. If I can't get that up soon then I'll PR removing the input altogether that way it's not messing stuff up.